frontend/src/components/pages/schemas/Schema.Details.tsx

```diff
@@ -166,121 +171,125 @@ class SchemaDetailsView extends PageComponent<{ subjectName: string }> {
+                                    <Tooltip label="Copy raw JSON to clipboard" placement="top" hasArrow={true} isDisabled={!isClipboardAvailable}>
+                                        <Button
+                                            isDisabled={!isClipboardAvailable}
+                                            onClick={() => {
+                                                navigator.clipboard.writeText(rawSchema).then(() => {
+                                                    toast({status: 'success', description: 'Schema copied to clipboard', duration: 1200});
+                                                })
+                                            }}
+                                        >
+                                            <Icon as={AiOutlineCopy} color="#555" width="18px"/>
+                                        </Button>
+                                    </Tooltip>
+                                </div>
+                            </NoClipboardPopover>
+
+                        </div>
+
+                        <div>
+                            {uiSettings.schemaDetails.viewMode == 'json' &&
+                                <KowlJsonView
+                                    shouldCollapse={false}
+                                    collapsed={false}
+                                    src={jsonViewObject}
+                                    style={{
+                                        border: 'solid thin lightgray',
+                                        borderRadius: '.25em',
+                                        padding: '1em 1em 1em 2em',
+                                        marginBottom: '1.5rem',
+                                    }}
+                                />
+                            }
+
+                            {uiSettings.schemaDetails.viewMode == 'fields' &&
+                                <Table
+                                    size="small"
+                                    columns={[
+                                        {title: 'Name', dataIndex: 'name', className: 'whiteSpaceDefault',}, // sorter: sortField('name')
+                                        {title: 'Type', dataIndex: 'type', className: 'whiteSpaceDefault', render: renderSchemaType}, //  sorter: sortField('type'),
+                                        {title: 'Default', dataIndex: 'default', className: 'whiteSpaceDefault'},
+                                        {title: 'Documentation', dataIndex: 'doc', className: 'whiteSpaceDefault'},
+                                    ]}
+                                    rowKey="name"
+                                    dataSource={fields}
+                                    pagination={false}
+                                    style={{
+                                        maxWidth: '100%',
+                                        marginTop: '1.5rem',
+                                        marginBottom: '1.5rem',
```

Comment:
If you use Box, you can just do `my="1.5rem"`

Reply:
@malinskibeniamin I was thinking I'd migrate these styles when migrating the table component?

Reply:
Makes sense, that sounds good to me, @rikimaru0345 any style guide to follow here?

Reply:
Both are fine, personally I'd use `my` if available.
This is not something worth spending time on / discussing. 
Both do the job, and changing between them is trivial